openapi-v1.yaml

```diff
@@ -5287,6 +5320,80 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /notebooks/{namespace}/{array}/copy:
+    parameters:
+      - type: string
+        name: namespace
+        in: path
+        required: true
+        description: The namespace of the notebook
+      - name: NotebookCopy
+        in: body
+        description: Input/Output information to create a new TileDB file
+        schema:
+          $ref: "#/definitions/NotebookCopy"
+        required: true
+      - name: X-TILEDB-CLOUD-ACCESS-CREDENTIALS-NAME
+        in: header
+        description: Optional registered access credentials to use for creation
+        type: string
+        required: false
+    post:
+      description: Copy a tiledb notebook at the specified location
+      tags:
+        - files
```

Comment:
Should this be in `files` package or `notebooks` ?

Reply:
`notebook` thanks